samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Can we title this heading as <strong>Step 1: Get the </strong><code><strong>ArcGIS StoryMap</strong></code><strong>  ....</strong></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done!</p>